global-api/importer/datasource_seeder/datasource_seeder.csv

```diff
@@ -0,0 +1,36 @@
+e6d533a4-0020-30bc-bbac-90627e499663;ClimateTRACE;ClimateTRACE Manure Management Emissions;Approach relies on the hypothesis that beef or daily facility area size can be used as a predictor to estimate the total cattle population, which can then be used to estimate manure management emissions.;third_party;globalapi;https://climatetrace.org/;EARTH;2020;2021;2021;annual;point source;en;;medium;;kg;https://github.com/climatetracecoalition/methodology-documents/blob/main/Agriculture/Agriculture%20sector-%20Enteric%20fermentation%20and%20Manure%20management%20(asset)%20Methodology.pdf;global_api;https://ccglobal.openearth.dev/api/v0/climatetrace/city/:locode/:year/:gpcReferenceNumber;V.1
+3b18e434-cd0e-3686-9537-6ae38ccb5c0c;ClimateTRACE;ClimateTRACE Enteric Fermentation Emissions;Approach relies on the hypothesis that beef or daily facility area size can be used as a predictor to estimate the total cattle population, which can then be used to estimate enteric fermentation emissions.;third_party;globalapi;https://climatetrace.org/;EARTH;2020;2021;2021;annual;point source;en;;medium;;kg;https://github.com/climatetracecoalition/methodology-documents/blob/main/Agriculture/Agriculture%20sector-%20Enteric%20fermentation%20and%20Manure%20management%20(asset)%20Methodology.pdf;global_api;https://ccglobal.openearth.dev/api/v0/climatetrace/city/:locode/:year/:gpcReferenceNumber;V.1
+3bfaac71-953d-354b-8e0c-dc3bb8ec34c3;Joint Research Centre;EDGARv7.0 Manufacturing Combustion Emissions;;third_party;globalapi;https://joint-research-centre.ec.europa.eu/index_en;EARTH;2021;2021;2021;annual;0.1 degree;en;;medium;;kg;https://edgar.jrc.ec.europa.eu/dataset_ghg70#intro;global_api;https://ccglobal.openearth.dev/api/v0/edgar/city/:locode/:year/:gpcReferenceNumber;I.3.1
+9e7138c0-510a-3f17-9464-c245842d9862;Joint Research Centre;EDGARv7.0 Road Transportation Emissions (No Resuspension);;third_party;globalapi;https://joint-research-centre.ec.europa.eu/index_en;EARTH;2021;2021;2021;annual;0.1 degree;en;;medium;;kg;https://edgar.jrc.ec.europa.eu/dataset_ghg70#intro;global_api;https://ccglobal.openearth.dev/api/v0/edgar/city/:locode/:year/:gpcReferenceNumber;II.1.1
+66403f84-41cf-4c24-8dd8-ae980e8ad687;deie_mendoza;Direcci�n de Estad�stica e Investigaciones Econ�micas;Electric energy users by tariff category, according to year. Municipal Statistical System based on data provided by Epre (Provincial Electrical Regulatory Entity);third_party;public;https://deie.mendoza.gov.ar/#!/;AR;2013;2022;2022;annual;city;en;;high;Emissions data calculated using the energy consumed methodology according to the sector proposed by GPC. AR5 emission factors for Argentina.;kg;https://deie.mendoza.gov.ar/#!/;global_api;https://ccglobal.openearth.dev/api/v0/deie_mendoza/city/:locode/:year/:gpcReferenceNumber;I.1.2

```

Comment:
There might be some issues with the encoding of special characters here. Can you replace these with the UTF-8 equivalents? Doesn't display correctly on Github so also probably not on our frontend.